documentation/cloud-manager-guide/environments/environment-details/node-details.md

```diff
@@ -0,0 +1,60 @@
+---
+keywords:
+title: Monitor Node Health
+description: Guide to review and monitor node status
+---
+# Monitor Node Health
+
+This guide outlines the steps required to monitor the status details for a specific node. The *Overview* tab provides a high-level view of all FID and Zookeeper node statuses for quick monitoring of cluster health. Each individual node also has a detailed view that provides further information on the status and health of a given node. The details provided for FID nodes versus Zookeeper nodes are slightly different and are outlined within this guide.
+
+## View node details
+
+There are two ways you can access the status details for a specific node. You can select the node name to open the node details dialog.
```

Comment:
Let's name both ways here, and then write them out individually with the image showing where to find it in the UI. Something like: 

"There are two ways to access the status details for a specific node: select the node name to open node details or select the options menu." 

You can select the node name from the list of available nodes to open the details dialog.

[image]

You can select the..."